hypervisor/dm/vpci/vhostbridge.c

```diff
@@ -43,8 +43,51 @@
  * @pre vdev != NULL
  * @pre vdev->vpci != NULL
  */
+/* The chart below shows the hostbridge DID high-byte of the platform later than broadwell, whose PCIEXBAR are always
```

Comment:
The commit message is wrong. What we want to do is to implement a full host bridge following HW spec. All the rest is a side effect or side gain.